system/logaccess/ElasticStack/ElasticStackLogAccess.hpp

```diff
@@ -35,9 +35,32 @@
 
 #include <cpr/response.h>
 #include <elasticlient/client.h>
+#include <elasticlient/scroll.h>
 
 using namespace elasticlient;
 
+class ElasticStackLogAccess;
+
+class ELASTICSTACKLOGACCESS_API ElasticStackLogStream : public CInterfaceOf<IRemoteLogAccessStream>
```

Comment:
encapsulation.  This class could be defined within the c++ file instead of the header.